scripts/deterministic/DeployScroll.s.sol

```diff
@@ -933,8 +1036,14 @@ contract DeployScroll is DeterminsticDeployment {
     }
 
     function deployL2ETHGateway() private {
+        address COUNTERPART;
+        if (ALTERNATIVE_GAS_TOKEN_ENABLED) {
```

Comment:
Why do we still need to deploy `L2ETHGateway`? Doesn't L1 ETH map to a standard ERC20 token on L2?

Reply:
If alternative gas token enabled, the counterpart of `L1GasTokenGateway` is `L2ETHGateway`.
if alternative gas token disabled,  the counterpart of `L1ETHGateway` is `L2ETHGateway`.
So we always need L2ETHGateway.